openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -83,6 +85,7 @@ def load(self, context, name, namespace, options):
             cmds.setAttr(standin_shape + ".dso", path, type="string")
             sequence = is_sequence(os.listdir(os.path.dirname(self.fname)))
             cmds.setAttr(standin_shape + ".useFrameExtension", sequence)
+            cmds.setAttr(standin_shape + ".abcFPS", 30)
```

Comment:
Wait a second - this doesn't sound right?

Reply:
I am not sure about this one, but it works for this specific animation. Let me know what you think

Reply:
Thank you for your reply, I will look it up more thoroughly !